iaf-aspose/src/main/java/nl/nn/adapterframework/extensions/aspose/services/conv/impl/convertors/Convertor.java

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Please add appropriate class heading comment referring to 'Integration Partners', to all classes